taxi/forms.py

```diff
@@ -0,0 +1,57 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+def validation_license_number(license_number):
+    length = 8
+    first_char = 3
+    last_char = 5

```

Comment:
Declare these values as constants and place them at the beginning of the file in the upper case. Besides that, make their naming more descriptive.